src/SignalRServiceExtension/SignalRMultiConnectionInfoAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Description;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    [AttributeUsage(AttributeTargets.ReturnValue | AttributeTargets.Parameter)]
+    [Binding]
+    public class SignalRMultiConnectionInfoAttribute : Attribute
```

Comment:
add descriptions. ///summary

Reply:
`SignalRConnectionInfoAttribute` doesn't have descriptions too.

Reply:
still need to add it.